polycommit/src/data_structures.rs

```diff
@@ -134,7 +137,7 @@ impl<F: Field> LabeledPolynomial<F> {
     }
 
     /// Evaluate the polynomial in `self`.
-    pub fn evaluate(&self, point: F) -> F {
+    pub fn evaluate(&self, point: &F) -> F {
```

Comment:
```suggestion
    pub fn evaluate(&self, point: F) -> F {
```
as long as `Field`s are still `Copy`